bin/db-initialization.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+import os
+
+from buildpg import V, asyncpg, render
+
+DB_NAME = os.environ["DATABASE_TO_CREATE"]
+
+
+def db_initialization():
+    async def setup():
+        user = os.environ["POSTGRES_DEFAULT_USERNAME"]
+        password = os.environ["POSTGRES_DEFAULT_PASSWORD"]
+        dsn = f"postgresql://{user}:{password}@postgres:5432/postgres"
+        conn = await asyncpg.connect(dsn=dsn)
```

Comment:
See https://www.postgresql.org/docs/current/libpq-envars.html. We should be using these environment variables alone to define the connection parameters. By leaving the connection parameters to environment variables in this way we automatically support all connection parameters supported by libpq _without code changes_.

These lines can then simply become:

```suggestion
        conn = await asyncpg.connect(database="")
```

Note that the definition of `database` here overrides anything set in the `PGDATABASE` env var, so we can use that var for the name of the database we want to create, which means we don't have to specify a "special" var (`DATABASE_TO_CREATE`) for this script.